megazords/ios-rust/MozillaTestServices/MozillaTestServicesTests/LoginsTests.swift

```diff
@@ -4,15 +4,13 @@
 
 @testable import MozillaTestServices
 
-import Glean
 import XCTest
 
 class LoginsTests: XCTestCase {
     var storage: LoginsStorage!
 
     override func setUp() {
         super.setUp()
-        Glean.shared.resetGlean(clearStores: true)
```

Comment:
Just out of curiosity, what's going in here and why can we get rid of it (instead of calling some other function more directly)

Reply:
Which probably means I can remove the import statement too :D